en/developer_guide/entity_list_ajax_pagination.rst

```diff
@@ -57,26 +57,155 @@ In order to make the pagination operational, the api should return a facade fill
 
 For instance, here comes the json returned by the api for listing the workflow functions:
 
-.. code-block:: JavaScript
-
-{
-    "links":{},
-    "collection_name":"workflow_functions",
-    "workflow_functions": [
-        {
-        "id":"558162b902b0cfc4118b45a7",
-        "links":{},
-        "name":"Contributor"
-        },
-        {
-        "id":"558162b902b0cfc4118b45a6",
+.. code-block:: json
+
+    {
         "links":{},
-        "name":"Validator"
-        }
-    ],
-    "recordsTotal":2,
-    "recordsFiltered":2
-}
+        "collection_name":"workflow_functions",
+        "workflow_functions": [
+            {
+            "id":"558162b902b0cfc4118b45a7",
+            "links":{},
+            "name":"Contributor"
+            },
+            {
+            "id":"558162b902b0cfc4118b45a6",
+            "links":{},
+            "name":"Validator"
+            }
+        ],
+        "recordsTotal":2,
+        "recordsFiltered":2
+    }
+
+Server side search
+------------------
+Open Orchestra provides a set of tools to help you perform the server side search.
+
+The ``OpenOrchestra\Pagination\MongoTrait\PaginationTrait`` trait which contains three methods: 
```

Comment:
you should say this trait is to be applied to a repository (supporting aggregation), so what you explain next is easier to understand